pkg/virt-operator/resource/apply/core.go

```diff
@@ -811,6 +820,26 @@ func (r *Reconciler) updateSynchronizationAddress() (err error) {
 		}
 		port = int32(p)
 	}
-	r.kv.Status.SynchronizationAddress = pointer.P(fmt.Sprintf("%s:%d", ip, port))
+	addresses := make([]string, len(ips))
+	for i, ip := range ips {
+		addresses[i] = fmt.Sprintf("%s:%d", ip, port)
+	}
+	r.kv.Status.SynchronizationAddresses = addresses
+	r.kv.Status.SynchronizationAddress = pointer.P(addresses[0])
+	return nil
+}
+
+func (r *Reconciler) getIpsFromAnnotations(pod *corev1.Pod) []string {
+	networkStatuses := multus.NetworkStatusesFromPod(pod)
+	for _, networkStatus := range networkStatuses {
+		if networkStatus.Interface == v1.MigrationInterfaceName {
+			if len(networkStatus.IPs) == 0 {
+				continue
```

Comment:
Please consider breaking the loop in this case, since the logic is only interested in a single interface.
```suggestion
				break
```

Reply:
My logic currently finds the first interface with an IP address, it will return once an IP address is found, not sure if the break is the correct thing to do

Reply:
Good point, yeah you are right.